docs/modules/ROOT/pages/documentation/microprofile/faulttolerance.adoc

```diff
@@ -21,9 +20,7 @@ detailed in the spec], the following is also implemented in Payara Server Commun
 === Configurable Executor Services
 The managed and scheduled managed executor pool can be configured via the _set-fault-tolerance-configuration_ asadmin command.
 
-The executor service is used to execute methods annotated with `@Asynchronous`, whereas the scheduled executor
-service is used by the CircuitBreaker and Timeout interceptors for their timeout operations (scheduling the circuit breaker
-to be set to half open, and just timing out respectively).
+The executor service is used to execute methods annotated with `@Asynchronous`, whereas the scheduled executor service is used by the CircuitBreaker and Timeout interceptors for their timeout operations (scheduling the circuit breaker to be set to half open, and just timing out respectively).
```

Comment:
@MattGill98, I'm a bit confused by this statement. Is the gist of the clarification that there are 2 managed executor services used to coordinate these 3 interceptors, a plain `ManagedExecutorService` for the `@Async` interceptor and a `ManagedScheduledExecutorService` to handle the other 2 correct? If this is the case it might be better to put this on a list and annotate the interceptor names and put them in `pre` format for better readibility

Reply:
What do you mean sorry? I haven't actually edited this line, but I think it reads fairly clearly